x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -22,82 +23,58 @@
  * {@link RestNodesCapabilitiesAction} and we use them to enable tests.
  */
 public class EsqlCapabilities {
-    /**
-     * Support for function {@code CBRT}. Done in #108574.
-     */
-    private static final String FN_CBRT = "fn_cbrt";
+    public enum Cap {
+        // Support for function {@code CBRT}. Done in #108574.
+        FN_CBRT(false),
```

Comment:
Should these be lowercase since the beginning? Yeah, it's ugly, but it would be handy

Reply:
I prefer it the way you did it, with capitalized enum values, and lower case registration.

Reply:
:+1: on capitalized values.

I'd probably make a second ctor that defaults to false - it's just so so so much more common. Usually I wouldn't do that for things, but for this I think it's worth it.